bin/node/runtime/src/lib.rs

```diff
@@ -1253,10 +1283,10 @@ impl_runtime_apis! {
 			sp_consensus_babe::BabeGenesisConfiguration {
 				slot_duration: Babe::slot_duration(),
 				epoch_length: EpochDuration::get(),
-				c: PRIMARY_PROBABILITY,
+				c: BABE_GENESIS_EPOCH_CONFIG.c,
 				genesis_authorities: Babe::authorities(),
 				randomness: Babe::randomness(),
-				allowed_slots: sp_consensus_babe::AllowedSlots::PrimaryAndSecondaryPlainSlots,
+				allowed_slots: BABE_GENESIS_EPOCH_CONFIG.allowed_slots,
```

Comment:
All constants values stated the same

Reply:
Could you add summary here?

Reply:
The Babe runtime module has [the current_epoch() and next_epoch() functions](https://github.com/paritytech/substrate/blob/93b231e79f5b4e551c34234e89fa4a2e5e9c1510/primitives/consensus/babe/src/lib.rs#L381-L386) that return the current and next epoch. 
Before the changes, they hadn't given any information about the configuration, which can potentially change between epochs. This changes fix this.